src/main/java/com/google/step/finscholar/data/Scholarship.java

```diff
@@ -0,0 +1,296 @@
+package com.google.step.finscholar.data;
```

Comment:
Insert Apache License header at the top.

Reply:
Get it! Now that I have access to the repo, I guess I'll resubmit a PR and delete the forked branch.